.github/workflows/dev-long-tests.yml

```diff
@@ -156,15 +156,30 @@ jobs:
         make libc6install
         CFLAGS="-O3 -m32" FUZZER_FLAGS="--long-tests" make uasan-fuzztest
 
-  clang-asan-ubsan-fuzz32:
+  clang-asan-fuzz32:
     runs-on: ubuntu-20.04
     steps:
     - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # tag=v4.1.1
-    - name: clang + ASan + UBSan + Fuzz Test 32bit
+    - name: clang + ASan + Fuzz Test 32bit
       run: |
         sudo apt-get -qqq update
         make libc6install
-        CC=clang CFLAGS="-O3 -m32" FUZZER_FLAGS="--long-tests" make uasan-fuzztest
+        CC=clang CFLAGS="-O3 -m32" FUZZER_FLAGS="--long-tests" make asan-fuzztest
+
+# The following test seems to have issues on github CI specifically,
+# it does not provide the `__mulodi4` instruction emulation
```

Comment:
Does it have issues before this PR, or only after? If it is the latter, this might also have issues in the kernel, and we would need to do something similar to what we do with `ZSTD_div64()`

https://github.com/facebook/zstd/blob/b880f20d52a925ebee373b5050c206ba325d935d/lib/common/zstd_deps.h#L82

Reply:
It looks like this might be a compiler bug.

Reply:
Yes, the issue doesn't reproduce on my local linux workstation.
I suspect a bug in the version of `ubsan` for `clang` currently deployed within github ci.

It's likely that this issue will be fixed at some unspecified point in the future.
We just can't wait for this issue to be fixed, the CI needs to continue running.
We'll re-enable this test when it can work properly on Github CI again.

Also, dropping temporarily the `ubsan` test for `clang` for 32-bit `x86` targets seems like a minor inconvenience, given that: 
- we mostly care about 64-bit `x64`
- `ubsan` 32-bit is still working fine with `gcc`
- we keep the more important `asan` test for 32-bit on `clang`